packages/@aws-cdk/core/lib/cfn-json.ts

```diff
@@ -43,7 +43,7 @@ export class CfnJson extends CoreConstruct implements IResolvable {
    * Normally there is no need to use this property since `CfnJson` is an
    * IResolvable, so it can be simply used as a value.
    */
-  private readonly value: Reference;
```

Comment:
See https://github.com/aws/aws-cdk/issues/13063 for some context no why this would have been needed if this fix was not introduced.